src/main/java/net/sacredlabyrinth/phaed/simpleclans/commands/clan/LeaderCommands.java

```diff
@@ -106,7 +111,7 @@ public void disband(Player player, ClanPlayer cp, Clan clan) {
             return;
         }
 
-        clan.disband(player, true, false);
+        new SCConversation(plugin, player, new DisbandPrompt()).begin();
```

Comment:
Since there is a confirmation via command, the GUI one could be removed, or it will be annoying for users.